hazelcast-client/src/main/java/com/hazelcast/client/connection/nio/ClientConnection.java

```diff
@@ -79,6 +80,7 @@ public ClientConnection(HazelcastClientInstanceImpl client,
         channel.attributeMap().put(ClientConnection.class, this);
         this.connectionId = connectionId;
         this.logger = client.getLoggingService().getLogger(ClientConnection.class);
+        this.startTime = System.currentTimeMillis();
```

Comment:
Flavor warning: 

I try to initialize fields when I can to reduce the size of the constructor.